third_party/3.6/click/decorators.pyi

```diff
@@ -29,6 +33,7 @@ def command(
     name: str = None,
     cls: type = Command,
     # Command
+    context_settings: Dict = None,
```

Comment:
A few things:
- From https://github.com/pallets/click/blob/master/click/decorators.py#L92 it looks like all parameters after cls are keyword-only.
- We prefer writing `context_settings: Optional[Dict] = ...` in typeshed. Also, it would be better to specify the type parameters for Dict (e.g. `Dict[str, int]`; haven't checked what context_settings actually needs).